datumaro/plugins/widerface_format.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, path, subset=None):
 
         if not subset:
             subset = osp.splitext(osp.basename(path))[0]
-            if re.fullmatch(r'wider_face_\S+_bbx_gt', subset):
+            if re.fullmatch(r'wider_face_\S+', subset):
```

Comment:
```suggestion
            if re.fullmatch(r'wider_face_\S+(_bbx_gt)|(_filelist)', subset):
```

Current variant will match any files, and the following `split()` will probably be wrong.